packages/xdl/src/EmbeddedAssets.ts

```diff
@@ -272,6 +272,10 @@ async function _maybeConfigureExpoUpdatesEmbeddedAssetsAsync(config: EmbeddedAss
       }
       return configPlist;
     });
+
+    // delete artifacts sometimes left behind by IosPlist util
+    fs.removeSync(path.join(supportingDirectory, 'Expo.json'));
+    fs.removeSync(path.join(supportingDirectory, 'Expo.plist.bak'));
```

Comment:
should be able to use https://github.com/expo/expo-cli/blob/6972d777022ce078fd8041517441894b23126052/packages/xdl/src/detach/IosPlist.ts#L95-L107 instead